src/Surfnet/StepupMiddlewareClient/Identity/Service/SecondFactorService.php

```diff
@@ -69,6 +69,19 @@ public function getVerified($secondFactorId)
         return $this->apiService->read('verified-second-factor/%s', [$secondFactorId]);
     }
 
+
+    /**
+     * @param string $secondFactorId
+     * @return null|array
+     * @throws AccessDeniedToResourceException When the consumer isn't authorised to access given resource.
+     * @throws ResourceReadException When the server doesn't respond with the resource.
+     * @throws MalformedResponseException When the server doesn't respond with (well-formed) JSON.
+     */
+    public function getVerifiedSkipVetting($secondFactorId)
```

Comment:
Like mentioned in previous PR's on MW and RA, the naming does not sound right. Also vetting is not skipped right? Only the proof of possession of the token step is skipped.

Reply:
I've changed it to be more informative regarding the prove of possession.

Reply:
Much better this way :+1: